### PR TITLE
Feat: #193 Move popup to right side

### PR DESCRIPTION
**Pull request by hoailinh1210**
### Summary (Please recap what you changed or fixed)
- Move the popup to the right side

image

### Checklist (If you won't pass this checklist please comment why)

-   [x] I removed all commented or unnecessary code.
-   [x] Provide the screenshots due to UI changed or bug fixed
-   [x] My code has covered these test cases (please list down your tested cases):

### CODE REVIEW

**Comment by djhahe** [comment]
@hoailinh1210  2 buttons is quite big, pls help to reduce the button size, single row is enough

**Comment by djhahe** [comment]
image

@hoailinh1210 pls check style on sign message popup

**Comment by djhahe** [comment]
image
Pls follow settings page styles, content should be on white background, also we need a confirm popup on removing site

File: client/src/services/ServiceWorker/Controllers/PopupController.js [hunk 1]
```diff
@@ -3,17 +3,17 @@ import browser from 'webextension-polyfill';
...
-
 const BLACKLIST_PROTOCOLS = ['chrome-extension:', 'chrome:'];
-
 const POPUP_TYPE = {
     SIGN_DEPLOY :  'dappSignDeployRequest',
     CONNECT_ACCOUNT: 'dappConnect',
     SIGN_MESSAGE: 'dappSignMessageRequest',
 }
+const NOTIFICATION_WIDTH = 367;
+const NOTIFICATION_HEIGHT = 720;
```
Comment (djhahe) [inline client/src/services/ServiceWorker/Controllers/PopupController.js:16]: pls use `width: 357px; height: 600px;` same with extension popup

**Review by djhahe — CHANGES_REQUESTED** [review]

**Comment by hoailinh1210** [comment]
Hi @djhahe, I updated 2 your comments, please take a look
> 



> image
> 
> @hoailinh1210 pls check style on sign message popup. I think we have enough space to show full message and pk

image



> image
> 
> Pls follow settings page styles, content should be on white background, also we need a confirm popup on removing site

image

**Comment by djhahe** [comment]
image
@hoailinh1210 please help to update styles. There are h and v scrolls.

**Review by djhahe — APPROVED** [review]

**Pull request merged**